lightning/src/offers/flow.rs

```diff
@@ -416,6 +416,26 @@ pub enum InvreqResponseInstructions {
 	},
 }
 
+/// Parameters for the reply path to a [`HeldHtlcAvailable`] onion message.
+pub enum HeldHtlcReplyPath {
+	/// The reply path to the [`HeldHtlcAvailable`] message should terminate at our node.
+	ToUs {
```

Comment:
It's too late now, but I think it would have been okay to just not implement the always-online sender to mostly-offline receiver case yet and focus purely on LSP clients.